ReflectometryServer/movement_strategy.py

```diff
@@ -25,7 +25,8 @@ def __init__(self, setup):
             setup (ReflectometryServer.geometry.PositionAndAngle):
         """
         self._angle = setup.angle
-        self._position_at_zero = Position(setup.y, setup.z)
+        self._initial_position_at_zero = Position(setup.y, setup.z)
+        self._current_position_at_zero = self._initial_position_at_zero
```

Comment:
this is a nice clean way of introducing the new variable and extensible if we want to add an angle to the motor axis